File: library/src/main/java/com/getstream/sdk/chat/model/enums/Pagination.java
package com.getstream.sdk.chat.model.enums;

public enum Pagination {
    greaterThan("id_gt"),
    greaterThanOrEqual("id_gte"),
    lessThan("id_lt"),
    lessThanOrEqual("id_lte");

    private String value;

    Pagination(final String value) {
        this.value = value;
    }

    public String get() {
        return value;
    }

    @Override
    public String toString() {
        return this.get();
    }
}

File: client/src/main/java/io/getstream/chat/android/client/api/models/Pagination.kt
package io.getstream.chat.android.client.api.models

enum class Pagination(private val value: String) {

    GREATER_THAN("id_gt"),
    GREATER_THAN_OR_EQUAL("id_gte"),
    LESS_THAN("id_lt"),
    LESS_THAN_OR_EQUAL("id_lte");

    fun get(): String {
        return value
    }

    override fun toString(): String {
        return this.get()
    }
}
